### DISCUSSION TITLE
Use workload identity in AKS

### DISCUSSION THREAD

**Discussion by Kapitan777** [microsoft/go-sqlcmd#551] [Q&A]
Is it possible to use last version for access to Azure SQL with AKS workload identity token? If yes could you show example of options?

**Comment by shueybubbles** [comment]
I think you can use `ActiveDirectoryDefault` authentication to use AKS workload identity. According to the `azidentity` package doc:
https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#DefaultAzureCredential

"[WorkloadIdentityCredential](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#WorkloadIdentityCredential), if environment variable configuration is set by the Azure workload identity webhook. "

Per the [AKS workload identity doc](https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview?tabs=dotnet#azure-identity-client-libraries) you don't need to do anything special in your code.
